static/css/legacy.less

```diff
@@ -1416,6 +1416,12 @@ tr {
       padding-left: 30px !important;
     }
   }
+  .column--aligned-middle{
+    vertical-align:"middle"
```

Comment:
no quotes on CSS properties.
`vertical-align:middle;` will work fine

Reply:
I had resolved this issue, can you check this now